src/libtorch.cc

```diff
@@ -77,7 +77,11 @@ class ModelState : public BackendModel {
   TRITONSERVER_Error* LoadModel(
       const std::string& artifact_name, const torch::Device device,
       std::string* model_path,
-      std::unique_ptr<torch::jit::script::Module>* torch_model);
+      std::shared_ptr<torch::jit::script::Module>* torch_model);
+
+  // Remove model from model map using its key
+  // Returns the number of models removed (0 or 1)
+  size_t UnloadModel(std::pair<bool, int64_t> key);
```

Comment:
If this will only return a 0 or 1 why not return a bool value instead? This bool would just mean if the model was unloaded or not.